### ISSUE TITLE
bundle-version: Prints Bundler's version information

### ISSUE THREAD

**Issue by tnir** [rubygems/bundler-site#849]
```suggestion
description: Prints Bundler's version information
```

How about this? Or, does "the bundler" convey anything else than "Bundler itself"?

_Originally posted by @olleolleolle in https://github.com/rubygems/bundler-site/pull/840#discussion_r934760841_

**Comment by tnir** [comment]
@olleolleolle Could you transfer this issue to rubygems/rubygems? I do not have permission to do it.

**Comment by olleolleolle** [comment]
> @olleolleolle Could you transfer this issue to rubygems/rubygems? I do not have permission to do it.

No, sorry, I don't have that access. Perhaps @deivid-rodriguez?
